Mathlib/Util/AssertExistsExt.lean

```diff
@@ -40,7 +40,7 @@ initialize assertExistsExt : SimplePersistentEnvExtension AssertExists (Std.Hash
 `addDeclEntry isDecl declName mod` takes as input the `Bool`ean `isDecl` and the `Name`s of
 a declaration or import, `declName`, and of a module, `mod`.
 It extends the `AssertExists` environment extension with the data `isDecl, declName, mod`.
-This information is used to capture declarations and modules that are required to not
+This information is used to capture declarations and modules that are required not to
 exist/be imported at some point, but should eventually exist/be imported.
```

Comment:
What was there previously sounded more correct to my ear, but maybe we should just sidestep this with:
```suggestion
This information is used to capture declarations and modules that are forbidden from
existing/being imported at some point, but should eventually exist/be imported.
```

Reply:
bors d+

Reply:
That wording is even nicer!